helm/falco/templates/falco-policy-exception.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: kyverno.io/v2alpha1
```

Comment:
We will need to gate this resource so it is only deployed if the `PolicyException` CRD exists

Reply:
Yes, I know this was mainly done for testing but now I can make it properly 😄